main.go

```diff
@@ -52,120 +45,66 @@ func main() {
 
 	dbConnectionString := fmt.Sprintf(
 		"host=%s port=%s user=%s dbname=%s sslmode=%s password=%s",
-		os.Getenv("WAUTH_DB_HOST"),
+		dbHost,
 		os.Getenv("WAUTH_DB_PORT"),
 		os.Getenv("WAUTH_DB_USER"),
 		os.Getenv("WAUTH_DB_NAME"),
 		os.Getenv("WAUTH_DB_SSLMODE"),
 		os.Getenv("WAUTH_DB_PASS"),
 	)
 
-	// Setup files
-	static, err := fs.Sub(content, "public/static")
-	if err != nil {
-		log.Fatalf("static files failed: %+v", err)
-	}
+	log.Printf("web-auth version: %s\n", Version)
 
-	log.Printf("web-auth version %s loaded", Version)
+	mailPort, _ := strconv.Atoi(os.Getenv("WAUTH_MAIL_PORT"))
 
-	port, err := strconv.Atoi(os.Getenv("WAUTH_MAIL_PORT"))
+	debug, err := strconv.ParseBool(os.Getenv("WAUTH_DEBUG"))
 	if err != nil {
-		log.Fatalf("failed to get port for mailer: %v", err)
+		debug = false
+	}
```

Comment:
NITPICK: Could just be `debug, _ := strconv.ParseBool(...)` because the zero value of `bool` is `false` - though you could argue that this is more explicit.

Reply:
Fixed